crates/sui-bridge/src/abi.rs

```diff
@@ -93,20 +93,27 @@ impl EthBridgeEvent {
         self,
         eth_tx_hash: ethers::types::H256,
         eth_event_index: u16,
-    ) -> Option<BridgeAction> {
-        match self {
+    ) -> BridgeResult<Option<BridgeAction>> {
+        Ok(match self {
             EthBridgeEvent::EthSuiBridgeEvents(event) => {
                 match event {
                     EthSuiBridgeEvents::TokensDepositedFilter(event) => {
                         let bridge_event = match EthToSuiTokenBridgeV1::try_from(&event) {
-                            Ok(bridge_event) => bridge_event,
+                            Ok(bridge_event) => {
+                                if bridge_event.sui_adjusted_amount == 0 {
+                                    return Err(BridgeError::ZeroValueBridgeTransfer(format!(
+                                        "Manual intervention is required: {}",
```

Comment:
out of curiosity, do we have a runbook for those manual action?

Reply:
no we do not yet